DocSum/tests/test_compose_on_gaudi.sh

```diff
@@ -161,19 +224,43 @@ function stop_docker() {
 }
 
 function main() {
-
+    echo "==========================================="
+    echo ">>>> Stopping any running Docker containers..."
     stop_docker
```

Comment:
There is already docker stop before the test.

Reply:
it is removed